app/scripts/components/exploration/atoms/atoms.ts

```diff
@@ -1,11 +1,15 @@
 import { atom } from 'jotai';
```

Comment:
Should the atoms used directly by the map (a common component) be defined together with the exploration atoms?

Reply:
Probably not. That's a good point, I'll move them.